docs/conf/atlantic/2023/writing-day.rst

```diff
@@ -67,3 +67,20 @@ together and offer each other some feedback on the important documentation
 we're using to move our careers forward.
 
 Let's help each other get ready for the job fair!
+
+The Blog Writing Room: Blogging for technical writers and documentarians
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Struggling to get into blogging? Our project, The Blog Writing Room, is designed to 
+give technical writers and documentarians a space to come together to work on writing 
+blog posts in a supportive and collaborative environment.
```

Comment:
Suggested edit to your second sentence: 

> "The Blog Writing Room is designed to create a supportive and collaborative environment for documentarians and tech writers to work on blog posts."

I would move your Writing Day goals under your first paragraph.